### PR TITLE
Inputs and search bar styles

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/modules/SearchBar/SearchBar.jsx [hunk 1]
```diff
@@ -18,8 +21,9 @@ const defaultProps = {
 const SearchBar = ({ discardPagination, placeholder, searchQuery, setSearchQuery }) => {
   return (
     <div className="j-search-bar">
+      <SearchIcon className="j-search-bar__icon" />
```
Comment (nik-kor) [inline src/modules/SearchBar/SearchBar.jsx]: you can use https://github.com/albburtsev/bem-cn to generate BEM class names

File: src/modules/forms/FormInput/FormInput.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  attachTo: PropTypes.string.isRequired,
+  component: PropTypes.string.isRequired,
+  disabled: PropTypes.bool,
+  label: PropTypes.string.isRequired,
+  type: PropTypes.string.isRequired,
+  normalize: PropTypes.func,
+  tooltip: PropTypes.string,
+};
+
+const renderField = (component, type, name, normalize, parse, disabled) => {
```
Comment (nik-kor) [inline src/modules/forms/FormInput/FormInput.jsx]: why not just to pass object with this properties?

File: src/modules/forms/FormInput/FormInput.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  );
+};
+
+const FormInput = props => {
+  const { attachTo, component, type, tooltip, normalize, parse, disabled } = props;
+
+  return (
+    <FormField>
+      <FormLabel htmlFor={attachTo} text={props.label} tooltip={tooltip} />
+      {renderField(component, type, attachTo, normalize, parse, disabled)}
```
Comment (nik-kor) [inline src/modules/forms/FormInput/FormInput.jsx]: or maybe just use Field component?
can't get the point of having additional function

File: src/modules/UI/icons/SearchIcon.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  className: PropTypes.string,
+};
+
+const SearchIcon = ({ className }) => {
+  return (
+    <div className={className}>
+      <svg width="24" height="24" viewBox="0 0 24 24">
```
Comment (nik-kor) [inline src/modules/UI/icons/SearchIcon.jsx]: why not use css?

Comment (nik-kor) [inline src/modules/UI/icons/SearchIcon.jsx]: what's the point of rendering svg with react?

Comment (ivanzusko) [inline src/modules/UI/icons/SearchIcon.jsx]: about inline styles - my bad, just copypaste from svg object. Replaced with `backgroud-image` property.

File: src/modules/forms/FormInput/FormInput.jsx (hunk 3 above)
Comment (ivanzusko) [inline src/modules/forms/FormInput/FormInput.jsx]: fixed

**Pull request merged**